app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java

```diff
@@ -332,6 +337,12 @@ public static void copyImageEvent() {
         TelemetryEvent.create(Category.ACTION, Method.COPY, Object.BROWSER_CONTEXTMENU, Value.IMAGE).queue();
     }
 
+    public static void addNewTabFromContextMenu() {
+        TelemetryEvent.create(Category.ACTION, Method.ADD, Object.TAB, Value.LINK)
```

Comment:
I'm uncertain here: to be comparable with shareLinkEvent() it should be
TelemetryEvent.create(Category.ACTION, Method.OPEN_TAB, Object.BROWSER_CONTEXTMENU, Value.LINK).queue();
but to be comparable with addNewTabFromHome() it should be
TelemetryEvent.create(Category.ACTION, Method.ADD, Object.TAB, Value.LINK).queue();
I think I'm favoring the first one?

Reply:
For easier quantitative comparison, do we focus on how user add a tab(add,tab,xxx)? Or which one is more popular when user long click on ContextMenu([share/copy/save],browser_contextmenu,[image/link])?

Reply:
That sounds like a product call but I think of UI pings more related to UI itself so comparing these are a different dash board which we select different pings from different UIs.